modules/ROOT/pages/aggregator/aggregator-examples.adoc

```diff
@@ -146,7 +146,7 @@ The flow also needs a listener for timeouts:
 <flow-ref name="whatToDoWithAllTheseNamesFromSomeCountry"/>
 ----
 
-Notice that the example does not configure an eviction time in the aggregator. That is because the default value is `0` (Evict right after the group is released), which is the desired behavior. So if a group completed or timed out, a subsequent value can be added to the group without being rejected.
+Notice that the example does not configure an eviction time in the aggregator. That is because the default value is `0`, which means the eviction takes place right after the group is released, which is the desired behavior. So if a group completed or timed out, a subsequent value can be added to the group without being rejected.
```

Comment:
Is "eviction time" = "timeout"? If so, choose just one term and use it throughout, to avoid confusion.

Reply:
For nonnative readers, "takes place" > "occurs"
```suggestion
Notice that the example does not configure an eviction time in the aggregator. That is because the default value is `0`, which means the eviction occurs directly after the group is released, which is the desired behavior. Therefore, if a group finishes or is timed out, a subsequent value can be added to the group without being rejected.
```